### PR TITLE
fix(launcher): retry /git/user api call on fail

### PR DESCRIPTION
**Pull request by vikram-raj**
Issue - https://github.com/openshiftio/openshift.io/issues/4743

### CODE REVIEW

**Comment by alien-ike** [comment]
### Ike Plugins (test-keeper)

Thank you @vikram-raj for this contribution!



It appears that no tests have been added or updated in this PR.

Automated tests give us confidence in shipping reliable software. Please add some as part of this change.

If you are an admin or the reviewer of this PR and you are sure that no test is needed then you can use the command `/ok-without-tests` as a comment to make the status green.


For more information please head over to official [documentation](http://arquillian.org/ike-prow-plugins/#_test_keeper_plugin). You can find there how to configure the plugin.

**Comment by vikram-raj** [comment]
[e2e-test-beta]
[e2e-test-released]

File: packages/fabric8-ui/src/app/space/app-launcher/services/app-launcher-gitprovider.service.ts [hunk 1]
```diff
@@ -87,28 +88,23 @@ export class AppLauncherGitproviderService implements GitProviderService {
         if (user && user.login) {
           this.gitHubUserLogin = user.login;
           const orgs: { [name: string]: string } = {};
-          return this.getUserOrgs(user.login).pipe(
```
Comment (christianvogt) [inline packages/fabric8-ui/src/app/space/app-launcher/services/app-launcher-gitprovider.service.ts]: Did you mean to remove this call? Will the user object always have the data we need?

Comment (vikram-raj) [reply packages/fabric8-ui/src/app/space/app-launcher/services/app-launcher-gitprovider.service.ts]: Yes, this API ```/api/services/git/user``` already return the GitHub organizations of the user. We use this API call to check if user git account is connected or not and if connected it returns all data that we need to show on the UI.

**Comment by vikram-raj** [comment]
[e2e-test-beta]
[e2e-test-released]

**Review by christianvogt — APPROVED** [review]

**Pull request merged**